ui/src/app/machines/views/MachineDetails/MachineNetwork/EditDHCP/__snapshots__/EditDHCP.test.tsx.snap

```diff
@@ -0,0 +1,980 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
A 1000 line snapshot seems a little excessive. If you think there's value in a snapshot here I would suggest targeting `FormikFormContent` instead of the whole wrapper, otherwise it captures a lot of the yup api.

Reply:
Thanks, looking at that test the snapshot probably isn't useful. I'll update it to test that the form exists.